### ISSUE TITLE
Won't intall with Luks selected

### ISSUE THREAD

**Issue by BrankdRad** [ChrisTitusTech/ArchTitus#273]
It won't install with Luks.  Always get stuck after partitioning and won't write to the encrypted volume. 

Thanks for all the work put in.
Branko

**Comment by h3artbl33d** [comment]
Yeah - the issue is that, after the `luksFormat` the system is still being strapped to `${partition3}`, rather than the `cryptdevice`. 

@ChrisTitusTech If luks is being used, the `${partition3}` invokes should be switched to `/dev/mapper/ROOT` in `scripts/0-preinstall.sh` on a number of lines.
